proposals/pubsub.md

```diff
@@ -0,0 +1,182 @@
+# PubSub enabled by default
+
+Authors:
+- @gozala
+
+Initial PR: https://github.com/protocol/web3-dev-team/pull/53
+
+<!--
+This template is for a proposal/brief/pitch for a significant project to be undertaken by a Web3 Dev project team.
+The goal of project proposals is to help us decide which work to take on, which things are more valuable than other things.
+-->
+<!--
+A proposal should contain enough detail for others to understand how this project contributes to our team’s mission of product-market fit
+for our unified stack of protocols, what is included in scope of the project, where to get started if a project team were to take this on,
+and any other information relevant for prioritizing this project against others.
+It does not need to describe the work in much detail. Most technical design and planning would take place after a proposal is adopted.
+Good project scope aims for ~3-5 engineers for 1-3 months (though feel free to suggest larger-scoped projects anyway). 
+Projects do not include regular day-to-day maintenance and improvement work, e.g. on testing, tooling, validation, code clarity, refactors for future capability, etc.
+-->
+<!--
+For ease of discussion in PRs, consider breaking lines after every sentence or long phrase.
+-->
+
+
+
+## Purpose &amp; impact 
+#### Background &amp; intent
+_Describe the desired state of the world after this project? Why does that matter?_
+<!--
+Outline the status quo, including any relevant context on the problem you’re seeing that this project should solve.
+Wherever possible, include pains or problems that you’ve seen users experience to help motivate why solving this problem works towards top-line objectives.
+-->
+
+As of this writing [pubsub][IPFS pubsub] is considered experimental and is disabled in go-ipfs.
+
+> Can be enabled by starting a daemon with `--enable-pubsub-experiment` flag
+
+In practice however many teams building products on IPFS use and depend on pubsub:
+
+- Textile nodes have pubsub enabled.
+- 3Box uses pubsub for log replication.
```

Comment:
```suggestion
- Ceramic uses pubsub for log replication.
```

@oed could you please

1. Check that I'm not misrepresenting.
2. Suggest any additional context that you think might be useful here.

Reply:
```suggestion
- Ceramic uses pubsub for queries and log replication.
```
I would just add this.